heron/statemgrs/src/java/com/twitter/heron/statemgr/FileSystemStateManager.java

```diff
@@ -182,6 +183,13 @@ public Lock getLock(String topologyName, LockName lockName) {
   }
 
   @Override
+  public ListenableFuture<TopologyMaster.MetricsCacheLocation> getMetricsCacheLocation(
```

Comment:
i did not find the method to delete all topo data. where is it? i find only the 'deleteTMasterLocation' and 'deleteTopology'. but they seems delete only one node instead of all data